### PR TITLE
Wording: Encourage contributing and PRs instead of forking the project on GitHub.

### LINKED ISSUE #540: Use 'GitHub' or 'Contribute on GitHub' instead of 'Fork on GitHub'.
My opinion of forking is often negative. Although a thing being forkable means the product is free as in freedom, it implies 'you should rather fork than contribute (by temporarily forking). To whom code is not a thing they come in contact with often, forking doesn't mean much; while contributing does.

### PR DESCRIPTION
**Pull request by jtagcat**
Closes #540

### CODE REVIEW

File: script/validate_json.rb [hunk 1]
```diff
@@ -24,8 +24,8 @@ module ExitCodes
 SupportedEntryKeys = ["difficulty", "domains", "email", "email_body", "email_subject", "meta", "name", "notes", "url"]
 SupportedLanguageKeys = ["about", "difficulty", "difficulty_easy", "difficulty_hard", "difficulty_impossible",
                         "difficulty_medium", "extension", "extensionguide", "extensionp1", "extensionp2",
-                        "extensionp3", "extensionp4", "extensionp5", "extensionp6", "footercredits", "fork",
-                        "forkproject", "guide", "guideeasy", "guideexplanations", "guidehard", "guideimpossible",
+                        "extensionp3", "extensionp4", "extensionp5", "extensionp6", "footercredits", "contribute",
+                        "pullrequest", "guide", "guideeasy", "guideexplanations", "guidehard", "guideimpossible",
                         "guidemedium", "hideinfo", "mikerogers", "name", "noinfo", "noresults", "noresultshelp",
                         "popular", "reset", "sendmail", "showinfo", "tagline", "title", "twitter", "whatisthis",
                         "whatisthis1", "whatisthis2", "whatisthis3", "whatisthis4"]
```
Comment (tupaschoal) [inline script/validate_json.rb:31]: Can you keep this one sorted, please?

File: _data/trans/en.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "extensionp5": "Chrome Web Store",
     "extensionp6": "",
     "footercredits": "Made by",
-    "fork": "Fork on GitHub",
-    "forkproject": "Fork the project on GitHub",
+    "contribute": "Contriute on GitHub",
```
Comment (tupaschoal) [inline _data/trans/en.json:17]: `contribute` needs to be moved to just after `about`
`pullrequest` to just after `popular`

This applies to all translations

File: _data/trans/en.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "about": "About",
+    "contribute": "Contriute on GitHub",
```
Comment (tupaschoal) [inline _data/trans/en.json:3]:
```suggestion
    "contribute": "Contribute on GitHub",
```

File: _data/trans/es.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
     "about": "Acerca de",
+    "contribute": "Fork en Github",
```
Comment (tupaschoal) [inline _data/trans/es.json:3]:
```suggestion
    "contribute": "Contribuya en Github",
```

File: _data/trans/pt_br.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
     "about": "About",
+    "contribute": "Faça fork no GitHub",
```
Comment (tupaschoal) [inline _data/trans/pt_br.json:3]:
```suggestion
    "contribute": "Contribua no GitHub",
```

File: _data/trans/pt_br.json [hunk 6]
```diff
@@ -29,6 +28,7 @@
     "noresults": "Não encontrou o que procurava?",
     "noresultshelp": "Ajude a tornar o JustDeleteMe melhor",
     "popular": "Popular",
+    "pullrequest": "Faça um Fork do projeto no GitHub",
```
Comment (tupaschoal) [inline _data/trans/pt_br.json:31]:
```suggestion
    "pullrequest": "Faça um 'Pull Request' no GitHub",
```

File: _data/trans/es.json [hunk 7]
```diff
@@ -29,6 +28,7 @@
     "noresults": "¿No encuentras lo que buscas?",
     "noresultshelp": "Ayúdanos a mejorar JustDeleteMe",
     "popular": "Popular",
+    "pullrequest": "Haz un Fork del proyecto en GitHub.",
```
Comment (tupaschoal) [inline _data/trans/es.json:31]:
```suggestion
    "pullrequest": "Haz un 'Pull Request' en GitHub.",
```

**Comment by jtagcat** [comment]
Would you mind translating the 2 changed keys, `contribute` and `pullrequest`? To quickly translate, you may click the link below:
@Hauwertlhaufn https://github.com/jdm-contrib/jdm/edit/master/_data/trans/de.json
@zernex201 https://github.com/jdm-contrib/jdm/edit/master/_data/trans/gr.json
@BraINstinct0 https://github.com/jdm-contrib/jdm/edit/master/_data/trans/kr.json
@AHOHNMYC https://github.com/jdm-contrib/jdm/edit/master/_data/trans/ru.json
@crazko https://github.com/jdm-contrib/jdm/edit/master/_data/trans/sk.json

Changes:
`"fork": "Fork on GitHub"` -> `"contribute": "Contribute on GitHub"`
`"forkproject": "Fork the project on GitHub"` -> `"pullrequest": "Make a Pull Request on GitHub"`

Thanks!

**Pull request merged**